docs/Metrics-System.md

```diff
@@ -51,20 +51,36 @@ The following shows the details of the available metrics.
 * CapacityTotal: Total capacity of the file system in bytes.
 * CapacityUsed: Used capacity of the file system in bytes.
 * CapacityFree: Free capacity of the file system in bytes.
+* FilesPinned: Total number of the files pinned.
```

Comment:
how about we have three sections: general (e.g. the capacity ones), logical operations (e.g the Files* ones), and RPC invocations (all the *ops) ones